### PR TITLE
Adding analytics test pages to performance task

### PR DESCRIPTION
**Pull request by micajuine-ho**
Created 2 analytics test pages based off real world pages, that test the visible trigger. Added these pages to the performance tasks' `config.json`.

/cc @calebcordry

### CODE REVIEW

File: build-system/tasks/performance/config.json [hunk 1]
```diff
@@ -8,7 +8,10 @@
     },
     {
       "handlerName": "analyticsHandler",
-      "urls": [],
+      "urls": [
+        "http://localhost:8000/test/manual/amp-analytics/performanceTestPages/performance-longer.html",
```
Comment (zhouyx) [inline build-system/tasks/performance/config.json:12]: visible-longer?

File: test/manual/amp-analytics/performanceTestPages/visible-longer.html [hunk 2]
```diff
@@ -0,0 +1,477 @@
+<!DOCTYPE html>
+<html lang="en" amp="" i-amphtml-layout="" transformed="self;v=1">
+
+<head>
+    <script async="" src="https://cdn.ampproject.org/v0.js"></script>
+    <script async="" custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+    <script async="" custom-element="amp-accordion" src="https://cdn.ampproject.org/v0/amp-accordion-0.1.js"></script>
```
Comment (zhouyx) [inline test/manual/amp-analytics/performanceTestPages/visible-longer.html:7]: nit: could you remove all unnecessary script

File: test/manual/amp-analytics/performanceTestPages/visible-shorter.html [hunk 3]
```diff
@@ -0,0 +1,470 @@
+<!DOCTYPE html>
```
Comment (zhouyx) [inline test/manual/amp-analytics/performanceTestPages/visible-shorter.html:1]: Could you please explain the difference between the two examples. Thanks

File: build-system/tasks/performance/config.json (hunk 1 above)
Comment (micajuine-ho) [reply build-system/tasks/performance/config.json:12]: ugh last minute name changes... thanks for the catch

File: test/manual/amp-analytics/performanceTestPages/visible-shorter.html (hunk 3 above)
Comment (micajuine-ho) [reply test/manual/amp-analytics/performanceTestPages/visible-shorter.html:1]: Added comments to the top of the files (and also renamed).
- `visible-basic` imitates a simple article with images and text 
- `visible-multiple` imitates a longer page with multiple blocks of text, images, and ads

File: build-system/tasks/performance/config.json (hunk 1 above)
Comment (zhouyx) [reply build-system/tasks/performance/config.json]: yeah it happens, a round of sanity check would be helpful.

File: test/manual/amp-analytics/performanceTestPages/visible-shorter.html (hunk 3 above)
Comment (zhouyx) [reply test/manual/amp-analytics/performanceTestPages/visible-shorter.html]: sounds good. thanks

**Pull request merged**